image/templates/helm/stackrox-secured-cluster/templates/admission-controller-scc.yaml

```diff
@@ -49,14 +49,34 @@ volumes:
 # "fake" document separator to aid GVK extraction heuristic
 ---
 {{- end }}
-
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why are we adding this in the admission controller SCC yaml file?

Reply:
Apparently admission controller uses the `anyuid` SCC and we deleted that from sensor, so I needed to restore it for the admission-controller to not introduce additional changes. But we could also check whether the admission-controller would be happy with `nonroot`.